templates/taxi/manufacturer_list.html

```diff
@@ -2,16 +2,18 @@
 
 {% block content %}
     <h1>Manufacturer List
-    </h1>
+    <a href="{% url 'taxi:manufacturer-create' %}">+</a>
+  </h1>
```

Comment:
Ensure that there are 2 whitespaces for indentation in your `.html` files as per the checklist requirement.